tools/pot/openvino/tools/pot/graph/node_utils.py

```diff
@@ -267,6 +267,19 @@ def get_lstm_ends(read_value, assigns, ignore_nodes):
     return lstm_outputs
 
 
+def create_node_name(input_node, mode=tuple):
+    """
+    Returns key for node input.
+    If input node has one output port -> key is name of input node.
+    Otherwise, key is tuple (input name, output port number)
+    """
+    key = input_node.fullname
+    if len(input_node.out_ports()) > 1:
+        port_number = input_node.in_port(0).get_source().out
+        key = (input_node.fullname, port_number) if mode == tuple else f"{input_node.fullname}.{port_number}"
+    return key
+
+
```

Comment:
Looks like we can reuse this function in https://github.com/openvinotoolkit/openvino/blob/9c82bfb5d3178cfb65875c52e2faf416a36a7e8a/tools/pot/openvino/tools/pot/graph/node_utils.py#L188

Reply:
Thanks. Will be updated.